### PR TITLE
Update graphicsmagick from 1.3.30 to 1.3.35

### PR DESCRIPTION
**Pull request by supechicken**
Tested on `x86_64`

### CODE REVIEW

File: packages/graphicsmagick.rb [hunk 1]
```diff
@@ -3,23 +3,10 @@
...
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/graphicsmagick-1.3.30-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/graphicsmagick-1.3.30-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: 'c2a653efa6a3ee47ef4cf50fc10965dd16e3acdee0baa3668ef2899d147cf6e4',
-     armv7l: 'c2a653efa6a3ee47ef4cf50fc10965dd16e3acdee0baa3668ef2899d147cf6e4',
-       i686: '5b44100564bf0e4910c9559cc4eca9a5ed3dea91ce3f47dfceb2bf885aa2d6c2',
-     x86_64: '8554b70d6ba8428089a26b61d1534874691d55fda6fedc6d69e79af53c3c7526',
-  })
+  source_url 'http://ftp.icm.edu.pl/pub/unix/graphics/GraphicsMagick/1.3/GraphicsMagick-1.3.35.tar.gz'
```
Comment (uberhacker) [inline packages/graphicsmagick.rb:8]: What is wrong with `https://downloads.sourceforge.net/project/graphicsmagick/graphicsmagick/1.3.35/GraphicsMagick-1.3.35.tar.xz`?

Comment (supechicken) [reply packages/graphicsmagick.rb:8]: `curl` will cause a redirecting loop when auto-selecting the download mirror in `sourceforge`

File: packages/graphicsmagick.rb [hunk 2]
```diff
@@ -30,8 +17,9 @@ class Graphicsmagick < Package
 
   def self.build
     system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}",
+           "--prefix=/usr/local",
+           "--libdir=/usr/local/lib64",
+           "--mandir=/usr/local/share/man",
```
Comment (uberhacker) [inline packages/graphicsmagick.rb]: We should be abstracting the install directories.  Does the following work?
```
system "./configure #{CREW_OPTIONS} \
--with-windows-font-dir=#{CREW_PREFIX}/share/fonts/truetype/msttcorefonts \
--with-perl=#{CREW_PREFIX}/bin/perl --disable-maintainer-mode"
```

File: packages/graphicsmagick.rb (hunk 1 above)
Comment (uberhacker) [reply packages/graphicsmagick.rb:8]: I didn't notice that when I tried.

File: packages/graphicsmagick.rb [hunk 3]
```diff
@@ -32,6 +19,7 @@ def self.build
     system './configure',
            "--prefix=#{CREW_PREFIX}",
            "--libdir=#{CREW_LIB_PREFIX}",
+           "--mandir=#{CREW_PREFIX}/share/man",
```
Comment (uberhacker) [inline packages/graphicsmagick.rb:22]: This line could be: `--mandir=#{CREW_MAN_PREFIX}` but you could probably replace all 3 lines above with `#{CREW_OPTIONS}`.

File: packages/graphicsmagick.rb (hunk 2 above)
Comment (supechicken) [reply packages/graphicsmagick.rb]: Done

File: packages/graphicsmagick.rb (hunk 1 above)
Comment (supechicken) [reply packages/graphicsmagick.rb:8]: Just checked, only using `https://sourceforge.net/projects/graphicsmagick/files/graphicsmagick/1.3.35/GraphicsMagick-1.3.35.tar.xz/download` will cause this issue

**Pull request merged**